hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/cli/RMAdminCLI.java

```diff
@@ -343,7 +343,14 @@ private int refreshQueues() throws IOException, YarnException {
     ResourceManagerAdministrationProtocol adminProtocol = createAdminProtocol();
     RefreshQueuesRequest request = 
       recordFactory.newRecordInstance(RefreshQueuesRequest.class);
-    adminProtocol.refreshQueues(request);
+    try {
+      adminProtocol.refreshQueues(request);
+    } catch (Exception e) {
+      System.out.println("Unable to issue queue refresh due to exception: "
```

Comment:
Hi @brucesw 
I have few comments.
1. can we change "Unable to issue queue refresh due to exception:" to "Unable to refresh queue due to exception:"?
2. Pls change "Queue refresh issued successfully." to "Queue was refreshed successfully."
Thanks

Reply:
Many thanks for the review
Messages have been changed accordingly